azure/application-gateway/advanced/variables.tf

```
@@ -197,11 +245,13 @@ variable "probe" {
       ]
     }])
   }))
-  default = [{
-    name = "Default"
-  }]
+  default = {
+    default_probe = {}
+  }
 }
 
+# WAF Variable
+
```

Default WAF mode is Prevention.  Should this be detection for a new AGW?
```suggestion
firewall_mode            = optional(string, "Detection")
```

---

Our design documents tend to request Detection. I've not yet seen an LLD that requests Prevention. Perhaps this could be discussed with the CSAs?